samples/SSHConfig/test.ssh-config

```diff
@@ -0,0 +1,78 @@
+# quick handy copy paste
+# PreferredAuthentications=password
```

Comment:
Each of these samples are identical copies of each other (and they're... well, very long).

How about splitting the one sample into two different files?

Reply:
I reduces samples to 2, and made one host per file